pom.xml

```diff
@@ -51,7 +51,7 @@
 		<dependency>
 			<groupId>com.xpandit.xray</groupId>
 			<artifactId>client-core</artifactId>
-    		<version>2.5.3</version>
+    		<version>2.6.0</version>
```

Comment:
Wasn't the client-core bumped to 2.6.0.1? Should we use that version instead?

Reply:
The version 2.6.0.1 will be used on my next PR after this one is approved

Reply:
Created PR #63 with the latest `client-core` version